api/include/mfxcommon.h

```diff
@@ -179,6 +179,8 @@ enum {
     MFX_PLATFORM_JASPERLAKE     = 32,
     MFX_PLATFORM_ELKHARTLAKE    = 33,
     MFX_PLATFORM_TIGERLAKE      = 40,
+    MFX_PLATFORM_ROCKETLAKE     = 42,
+    MFX_PLATFORM_ALDERLAKE_S    = 43,
```

Comment:
Please, update manual.

Reply:
hi dvrogozh, i do not find any manual to notify the user about the new platform as well as the new platform enabling does not need the user to do any change on application or platform configuration.

Reply:
You need to update this section in the manual:
https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/mediasdk-man.md#platformcodename

It explicitly lists platform codenames.

Reply:
done. thank you!